edgelet/build/linux/package.sh

```diff
@@ -60,37 +60,36 @@ if [ -z "$DOCKER_IMAGE" ]; then
 fi
 
 case "$PACKAGE_ARCH" in
-    'amd64')
-        MAKE_FLAGS="DPKGFLAGS='-b -us -uc -i'"
-        ;;
+'amd64')
+    MAKE_FLAGS="DPKGFLAGS='-b -us -uc -i'"
+    ;;
 
-    'arm32v7')
-        RUST_TARGET='armv7-unknown-linux-gnueabihf'
-        ;;
+'arm32v7')
+    RUST_TARGET='armv7-unknown-linux-gnueabihf'
+    ;;
 
-    'aarch64')
-        RUST_TARGET='aarch64-unknown-linux-gnu'
-        ;;
+'aarch64')
+    RUST_TARGET='aarch64-unknown-linux-gnu'
+    ;;
 esac
 
 if [ -n "$RUST_TARGET" ]; then
     RUST_TARGET_COMMAND="rustup target add $RUST_TARGET &&"
 fi
 
-
 case "$PACKAGE_OS.$PACKAGE_ARCH" in
-    centos7.amd64)
-        SETUP_COMMAND=$'
+centos7.amd64)
+    SETUP_COMMAND=$'
```

Comment:
In general, it looks like the formatter aligned match arms with the outer case statement. But in situations like this one, where we use $-quoting, the string contents are left at their old indentation. I see why the formatter doesn't want to mess with string contents, but it looks weird. I'd suggest we either (a) find all the $-quoted command strings and back them up four spaces, or (b) don't format this file in this PR. I'd prefer (b), assuming it won't fail our checkin gate.

Reply:
Removed changes to this file